src/seedu/addressbook/Main.java

```diff
@@ -80,6 +82,12 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
+            try {
+                checkStorageFile();
+            } catch (StorageFileNotFoundException e) {
+                e.printStackTrace();
```

Comment:
Very bad way of handling exceptions. Printing the stracj trace for exception handling is not acceptable. Show a warning to the user.